### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Authorization_Colour Contrast: Colour Contrast Ratio for place-holder text at "Specific Users" and "Roles or Groups" list items is less than 4.5:1 

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#120]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #ColorContrast; #MAS1.4.3

**Environment Details:**
URL: https://manage.iis.net/webserver/authorization
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome 68.0.3440.106
MAS ID Violated: MAS 1.4.3

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till Authorization and press enter
6. Tab till more button(...) and press enter
7. Tab till edit button and press enter
8. Tab till "Specific Users" and "Roles or Groups" list items and press enter. Verify colour contrast ratio for place-holder text is >= 4.5:1

**Actual Result:**
Colour contrast ratio for place holder text at "Specific Users" and "Roles or Groups" list items is < 4.5:1

**Expected Result:**
Colour contrast ratio for place holder text at "Specific Users" and "Roles or Groups" list items should be >= 4.5:1

**User Impact:**
Users with disabilities finds its difficult in working through the application if colour contrast ratio is < 4.5:1

**Note:**
On enabling "Use Specific HTTP Methods" toggle button, we are getting some text field having place holder text. Colour contrast ratio for place holder text is less than 4.5:1
This issue has been observed in the pages which has place-holder text

**Attachments:**
![bug120_web server_authorization_colour contrast1](https://user-images.githubusercontent.com/42638450/44704952-d04ac480-aaba-11e8-8cfe-89b17746c4a5.png)
![bug120_web server_authorization_colour contrast2](https://user-images.githubusercontent.com/42638450/44704954-d04ac480-aaba-11e8-9ca7-df4f220ed5d3.png)
![bug120_web server_authorization_colour contrast3](https://user-images.githubusercontent.com/42638450/44723882-dc507980-aaee-11e8-8c45-9ecfa41166aa.png)

![bug 120_color contrast](https://user-images.githubusercontent.com/42638450/44909874-aa355680-ad3e-11e8-81cb-a4df36276d31.png)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
